scripts/build.sh

```diff
@@ -0,0 +1,47 @@
+# Create a browser extension release package
+
+# 1. Capture browser type from argument
+browserType=$1
+
+# 2. Construct release zip filename
+zipName=$browserType-release.zip
+
+# 3. Ensure release directory exists (create if needed)
+mkdir -p build/release/
+
+# 4. Clean up any existing release zip
+rm -f build/release/$browserType-release.zip
```

Comment:
Do you need to force this? Scary stuff.

Reply:
> Do you need to force this? 

Not really, I used the `-f` flag to suppress the exit code when the file couldn't be found. However, I've modified the command to handle missing files differently. Thanks for your comment.